pkg/epp/scheduling/scheduler_config.go

```diff
@@ -31,3 +40,48 @@ type SchedulerConfig struct {
 	profileHandler framework.ProfileHandler
 	profiles       map[string]*framework.SchedulerProfile
 }
+
+func LoadSchedulerConfig(theConfig *v1alpha1.EndpointPickerConfig, references map[string]plugins.Plugin,
```

Comment:
can we pass only the SchedulingProfiles section of the config?
or is there a good reason that we pass the whole config struct?

Reply:
Done